solutions/silver/cf-1000C.mdx

```diff
@@ -24,36 +30,29 @@ This allows us to apply prefix sums on each point in our compressed number line
 
 ```cpp
 #include <bits/stdc++.h>
-
 using namespace std;
-using ll = long long;
-using vi = vector<int>;
-using pi = pair<int, int>;
 
-#define f first
-#define s second
-#define all(x) begin(x), end(x)
+using ll = long long;
 
 int main() {
 	int n;
 	cin >> n;
 	// This stores the original endpoints to use in our frequency array later
 	vector<pair<ll, ll>> segments(n);
-	// This stores only the unique endpoints (also, the endpoints will be in
-	// sorted order).
+	// This stores only the unique endpoints in sorted order
 	set<ll> points;
 	for (int i = 0; i < n; i++) {
-		cin >> segments[i].f >>
-		    segments[i].s;  // Read in the endpoints of the i-th segment
-		// This ensures that only the unique points are stored
-		points.insert(segments[i].f);
-		points.insert(segments[i].s + 1);
+		cin >> segments[i].first >> segments[i].second;
+		// In a set, only unique points will be stored
```

Comment:
damn, that's crazy

Reply:
xd do a git blame ima not the person to blame for this
![image](https://github.com/cpinitiative/usaco-guide/assets/44814054/845d9d68-9126-4bec-8d84-a941bc4b1a4d)